massa-event-cache/src/config.rs

```diff
@@ -0,0 +1,16 @@
+use std::path::PathBuf;
+
+pub struct EventCacheConfig {
+    /// Path to the hard drive cache storage
+    pub event_cache_path: PathBuf,
+    /// Maximum number of entries we want to keep in the event cache
+    pub max_event_cache_length: usize,
+    /// Amount of entries removed when `event_cache_size` is reached
+    pub snip_amount: usize,
+    /// Maximum length of an event data (aka event message)
+    pub max_event_data_length: u64,
+    /// Thread count
+    pub thread_count: u8,
+    /// Call stack max length
+    pub max_recursive_call_depth: u16,
```

Comment:
WARNING: sometimes the call stack can be HIGHER than the max recursive call depth. Example: at the root level (recursive depth = 1) the call stack contains 2 elements. So at MAX_DEPTH, it would contain MAX_DEPTH + 1 elements. I'm not a big fan of the coupling being created here between very distant features.

Not sure how to improve this.
What do you think ?

Reply:
I've renamed the field to max_call_stack_length to avoid the confusion. It is now not limited (u16::MAX) but a note mention the assumption (in massa node). What do you think?